gallery/src/components/NxModal/NxModalStackedExample.tsx

```diff
@@ -9,15 +9,29 @@ import React, {useState} from 'react';
 import {NxModal, NxFontAwesomeIcon} from '@sonatype/react-shared-components';
 import {faAngry} from '@fortawesome/free-solid-svg-icons';
 
+/* eslint-disable */
 export default function NxModalStackedExample() {
   const [showModal, setShowModal] = useState(false);
   const [showModal2, setShowModal2] = useState(false);
+  const [modalStack] = useState([]);
```

Comment:
We use `useState` so that the variable isn't re-initialized every time this function-component is called.